File: src/test/java/com/foursquare/pilgrim/PilgrimSdk.java
package com.foursquare.pilgrim;

import android.content.Context;

public class PilgrimSdk {

    public static void with(PilgrimSdk.Builder builder) {
        //do nothing
    }


        public static class Builder {
            public Builder(Context context) {

            }

            public Builder consumer(String a, String b) {
                return this;
            }

            public Builder logLevel(LogLevel logLevel) {
                return this;
            }

            public Builder enableDebugLogs() {
                return this;
            }
        }
}

File: src/test/kotlin/com/foursquare/pilgrim/PilgrimSdk.kt
package com.foursquare.pilgrim

import android.content.Context

class PilgrimSdk {

    companion object {
        fun with(builder: Builder?) {
            //do nothing
        }
    }


    class Builder(context: Context?) {
        fun consumer(a: String?, b: String?): Builder {
            return this
        }

        fun logLevel(logLevel: LogLevel?): Builder {
            return this
        }

        fun enableDebugLogs(): Builder {
            return this
        }
    }
}
